tests/test_services.py

```
@@ -0,0 +1,351 @@
+from types import SimpleNamespace
```

Ruff (0.12.2)

Missing docstring in public module

(D100)